admin-ui/plugins/auth-server/redux/sagas/OIDCSaga.js

```diff
@@ -45,6 +47,21 @@ function* newFunction() {
   return new OIDCApi(api)
 }
 
+function* newUMAFunction() {
```

Comment:
please create a separate saga for uma resource.

Reply:
done, updated @duttarnab